### PR TITLE
(#1026) Update example page content

### LINKED ISSUE #1026: Update sample pages on example site and storybook with content from cgov-digital-platform
### Change Request Description
The requested change is to update the Example site so that it matches the ODE exactly to improve testing. The pages to be updated on the Example site and their equivalent on the ODE are listed below.

### Expected Change
Update the following on the Example Site (https://designsystem-dev.cancer.gov/develop/example-site/)

- [x] Make the [CancerGov Home Page](https://designsystem-dev.cancer.gov/develop/example-site/example-pages/cgov-home.html) match the [ODE Home Page](https://www-dev-ac.cancer.gov/)
- [x] Make the [Landing Page](https://designsystem-dev.cancer.gov/develop/example-site/example-pages/landing-page.html) match the [Grants and Training Landing page](https://www-dev-ac.cancer.gov/grants-training)
- [x] Create new Fake Landing Page 1 to match the [ODE Sample Page 1](https://www-dev-ac.cancer.gov/sample-page-1)
- [x] Create new Fake Landing Page 2 to match the [ODE Sample Page 2](https://www-dev-ac.cancer.gov/sample-page-2)
  * Update to use a 3 guide card row (choose link titles as you wish)
- [x] Create new Fake Landing Page 3 to match the [ODE Sample Page 3](https://www-dev-ac.cancer.gov/sample-page-3)
  * Update to remove the title
  * Update the hero to be a dark hero with primary CTA
  * Update the second promo block to have image right
- [x] Create new Fake Landing Page 4 to match the [ODE Sample Page 4](https://www-dev-ac.cancer.gov/sample-page-4)
  * Update to use a 3 guide card row (choose link titles as you wish)

### Acceptance Criteria
```gherkin
Scenario: The DS Example Site matches the test pages on the ODE
Given a user is viewing the DS Example Site
Then the DS Example Site exactly matches the test pages on the ODE
```

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1026

Updating content on example pages to match active site

### CODE REVIEW

**Review by olitharp-nci — CHANGES_REQUESTED** [review]
Sorry, looks like Imma be blocking this until we get the image situation cleared up 🎉

**Review by sarinapadilla — APPROVED** [review]
Looks good!

**Comment by arcepaul** [comment]
Please perform axe Dev Tools scans on the pages.  A new issue is popping up from a duplicated div in the footer.  Look for the id `email-error-message` and it comes up twice.  The div should  not be there at all since I did not try to submit an email.

**Comment by arcepaul** [comment]
On the promoblocks page, Alter the second title `NCI Annual Plan & Budget Proposal for Fiscal Year 2024 (Dark)`. The `<h2>`s are being used as aria labels and are getting flagged as not being unique any more.

**Comment by arcepaul** [comment]
Hero images on https://designsystem-dev.cancer.gov/pr-1065/example-site/example-pages/nci-hero.html are now missing

**Review by olitharp-nci — CHANGES_REQUESTED** [review]
Can you change every instance of "promoblock" to "promo block" and remove "nci" from "nci hero"

File: testing/ncids-js-testing/public/example-pages/promoblock-example.html
Comment (olitharp-nci) [inline testing/ncids-js-testing/public/example-pages/promoblock-example.html:1]: rename to promo-block-example

File: testing/ncids-js-testing/public/example-pages/nci-cta-strip.html
[the remark this answers is not in the document]
Comment (karlikpj) [reply testing/ncids-js-testing/public/example-pages/nci-cta-strip.html:1]: updated and also added a comment in the commits for this/hero/promo block updates.

**Review by arcepaul — APPROVED** [review]

**Pull request merged**